### PR TITLE
Remote Environment Controller Improvement

### PR DESCRIPTION
**Pull request by Szymongib**
**Description**

Changes proposed in this pull request:

- Added ability to configure synchronization period
- Setting Helm release status on RE resource

**Related issue(s)**
#1155

### CODE REVIEW

**Comment by Szymongib** [comment]
Do not merge before https://github.com/kyma-project/kyma/pull/1474

File: components/remote-environment-controller/README.md [hunk 1]
```diff
@@ -10,5 +10,19 @@ Remote Environment Controller detects changes in Remote Environment custom resou
...
  - Remote Environment deleted - Controller deletes Helm chart corresponding to the given RE.
- 
\ No newline at end of file
+
+ 
+ ## Usage
+ 
+ The Remote Environment Controller has the following parameters:
+ - **appName** - This is the name used in controller registration. The default value is `remote-environment-controller`.
+ - **domainName** - Domain name of the cluster. Default domain name is `kyma.local`.
```
Comment (tomekpapiernik) [inline components/remote-environment-controller/README.md]:
```suggestion
 - **domainName** - Domain name of the cluster. The default domain name is `kyma.local`.
```

File: components/remote-environment-controller/README.md [hunk 2]
```diff
@@ -10,5 +10,19 @@ Remote Environment Controller detects changes in Remote Environment custom resou
...
+ - **namespace** - Namespace where the Remote Environment charts will be deployed. The default namespace is `kyma-integration`.
+ - **tillerUrl** - Tiller release server url. The default is `tiller-deploy.kube-system.svc.cluster.local:44134`.
```
Comment (tomekpapiernik) [inline components/remote-environment-controller/README.md]:
```suggestion
 - **tillerUrl** - Tiller release server URL. The default is `tiller-deploy.kube-system.svc.cluster.local:44134`.
```

File: components/remote-environment-controller/README.md [hunk 3]
```diff
@@ -10,5 +10,19 @@ Remote Environment Controller detects changes in Remote Environment custom resou
...
+ - **syncPeriod** - Time period between resyncing existing resources. The default value is `30` seconds.
+ - **installationTimeout** - Time after the release installation will time out. The default value is `240` seconds.
+ - **proxyServiceImage** - The image of the Proxy Service that will be used in Remote Environment Chart.
```
Comment (tomekpapiernik) [inline components/remote-environment-controller/README.md]:
```suggestion
 - **proxyServiceImage** - Proxy Service image version to be used in the Remote Environment chart.
```

File: components/remote-environment-controller/README.md [hunk 4]
```diff
@@ -10,5 +10,19 @@ Remote Environment Controller detects changes in Remote Environment custom resou
...
+ - **eventServiceImage** - The image of the Event Service that will be used in Remote Environment Chart.
```
Comment (tomekpapiernik) [inline components/remote-environment-controller/README.md]:
```suggestion
 - **eventServiceImage** - Event Service image version to be used in the Remote Environment chart.
```

File: components/remote-environment-controller/README.md [hunk 5]
```diff
@@ -10,5 +10,19 @@ Remote Environment Controller detects changes in Remote Environment custom resou
...
+ - **eventServiceTestsImage** - The image of the Event Service Tests that will be used in Remote Environment Chart.
```
Comment (tomekpapiernik) [inline components/remote-environment-controller/README.md]:
```suggestion
 - **eventServiceTestsImage** - Event Service Tests image version to be used in the Remote Environment chart.
```

**Pull request merged**